http-server/src/server.rs

```diff
@@ -281,7 +287,8 @@ impl Server {
 						} else {
 							collect_batch_response(rx).await
 						};
-						tracing::debug!("[service_fn] sending back: {:?}", &response[..cmp::min(response.len(), 1024)]);
+						tracing::debug!(tx_batch_call_len = response.len());
```

Comment:
We can still have a log message in here right? `tracing::debug!("[service_fn] sending back response.", tx_batch_call_len = response.len());` or something?